docker/.env.example

```diff
@@ -6,7 +6,7 @@ LOG_PATH=/root/.flowise/logs
 BLOB_STORAGE_PATH=/root/.flowise/storage
 
 # NUMBER_OF_PROXIES= 1
-# CORS_ORIGINS="*"
+# CORS_ORIGINS=*
 # IFRAME_ORIGINS="*"
```

Comment:
probably the same with `IFRAME_ORIGINS`

Reply:
Will rectify and have a new PR

Reply:
you can add the changes on this PR too, dont have to be a new one

Reply:
Yeah changed, let me know if there are anyother